.github/workflows/docs-pr.yml

```diff
@@ -4,6 +4,8 @@ name: "Docs / Build PR"
 
 on:
   push:
+    branches:
+      - master
     paths:
```

Comment:
Commit: "docs: format workflow files "

This change belongs to previous commit (the one updating the conditions).

Reply:
moved to a proper commit.